assembl/static2/css/components/ideas.scss

```diff
@@ -0,0 +1,104 @@
+.ideas-level {
+    margin: 0 auto;
+    &.animating-towards-inline, &.is-inline {
+        position: relative;
+
+        // scroll containers are an empty area around the scroll button, so that user does not mis-click on idea link which is next to this button
+        .scroll-left-container, .scroll-right-container {
+            display: inline-block;
+            height: 80px;
+            width: 80px;
+            position: absolute;
+            top: calc(50% - 40px);
+            background-color: rgba(0,0,0,0);
+        }
+        .scroll-left-container {
+            left: 0;
+        }
+        .scroll-right-container {
+            right: 0;
+        }
+
+        .scroll-left, .scroll-right {
+            display: none;
+            position: absolute;
+            width: 50px;
+            height: 50px;
+            background-color: rgba(195, 195, 195, 0.6);
+            top: calc(50% - 25px);
+            cursor: pointer;
+            &:before {
+                font-size: 42px;
+                color: #000;
+                width: 100%;
+                text-align: center;
+                position: absolute;
+            }
+        }
+        .scroll-left {
+            z-index: 1000;
+            left: 0;
+            &:before {
+                content: "<";
+            }
+        }
+
+        &:hover {
+            .scroll-left, .scroll-right {
+                display: block;
+            }
+        }
+        .scroll-right {
+            z-index: 1001;
+            right: 0;
+            &:before {
+                content: ">";
```

Comment:
Yes, this is a first pass for the appearance of the scroll buttons. Next iterations will probably need fontello or an image.